dags/nci_ard/nci_fetch_brdf.py

```diff
@@ -51,13 +51,8 @@
 params["umask"] = " umask 022 "
 
 if is_production_run:
-
     # Production runs
-    # schedule_interval = "30 0 * * *"
-
-    # Disable this dag until we are absolutely happy with it
-    # todo = remove this line when it's ready to for review
-    schedule_interval = None
+    schedule_interval = "31 05,11,17,23 * * *"
```

Comment:
```suggestion
    schedule_interval = "30 23 * * *"
```
Let's keep to the current schedule

Reply:
right.. I extracted the original crontab line into https://gajira.atlassian.net/browse/DSNS-115 way back . Did not know the schedule changed. Anyway, yes , will use this one :)